api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -106,6 +106,24 @@ enum DiscoverResource {
   DISCOVER_RESOURCE_DATABASE_POSTGRES_GCP = 15;
   DISCOVER_RESOURCE_DATABASE_MYSQL_GCP = 16;
   DISCOVER_RESOURCE_DATABASE_SQLSERVER_GCP = 17;
+
+  DISCOVER_RESOURCE_DATABASE_POSTGRES_REDSHIFT_SERVERLESS = 18;
+  DISCOVER_RESOURCE_DATABASE_POSTGRES_AZURE = 19;
+  DISCOVER_RESOURCE_DATABASE_DYNAMODB = 20;
+  DISCOVER_RESOURCE_DATABASE_CASSANDRA_KEYSPACES = 21;
+  DISCOVER_RESOURCE_DATABASE_CASSANDRA_SELF_HOSTED = 22; // Cassandra & ScyllaDb
+  DISCOVER_RESOURCE_DATABASE_ELASTICSEARCH_SELF_HOSTED = 23;
+  DISCOVER_RESOURCE_DATABASE_REDIS_ELASTICACHE = 24; // Elasticache & MemoryDb
+  DISCOVER_RESOURCE_DATABASE_REDIS_MEMORYDB = 25;
+  DISCOVER_RESOURCE_DATABASE_REDIS_AZURE_CACHE = 26;
+  DISCOVER_RESOURCE_DATABASE_REDIS_CLUSTER_SELF_HOSTED = 27;
+
+  DISCOVER_RESOURCE_DATABASE_MYSQL_AZURE = 28;
+  DISCOVER_RESOURCE_DATABASE_SQLSERVER_AZURE = 29;
+  DISCOVER_RESOURCE_DATABASE_SQLSERVER_MICROSOFT = 30;
+  DISCOVER_RESOURCE_DATABASE_COCKROACHDB_SELF_HOSTED = 31;
+  DISCOVER_RESOURCE_DATABASE_MONGODB_ATLAS = 32;
+  DISCOVER_RESOURCE_DATABASE_SNOWFLAKE = 33;
```

Comment:
I was confused how I should be constructing these event names. Most of them followed this format:

`DISCOVER_RESOURCE_DATABASE_<PROTOCOL>_<LOCATION>`

i didn't know how to define the location for `dynamodb` and `snowflake` so i didn't define the location. For example do i tack on `AWS` for `dynamoDb`? (obvious it's aws?)

Reply:
I think for DynamoDB it is ok to not mention AWS.

I'm not sure about snowflake, but it seems we support their managed version which can be hosted on AWS but is not actually managed by AWS.
Maybe we can should keep your current version and come back to it if something changes.
It would be awkward to have `DISCOVER_RESOURCE_DATABASE_SNOWFLAKE_SNOWFLAKECOMPUTING` (?)

As for the format, yeah, I think that's the implicit pattern we are following.